src/ansible_navigator/actions/_actions.py

```diff
@@ -52,54 +59,95 @@ def _import_all(package: str) -> None:
 
 
 def register(cls: Any) -> Any:
-    """Decorator for registering a new action"""
+    """Register an action, used as a decorator.
+
+    :param cls: The class to register
+    :returns: The class after registration
+    """
     package, _, action = cls.__module__.rpartition(".")
     pkg_info = _ACTIONS.setdefault(package, {})
     pkg_info[action] = ActionT(name=action, cls=cls, kegex=re.compile(cls.KEGEX))
     return cls
 
 
 def get(package: str, action: str) -> Callable:
-    """Get a given action"""
+    """Import and return a given action.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :returns: The action's registered class
+    """
     _import(package, action)
     return _ACTIONS[package][action].cls
 
 
 def get_factory(package: str) -> Callable:
-    """Create a get() function for one package"""
+    """Create a ``get`` function for one package.
```

Comment:
Maybe this would be more accurate?
```suggestion
    """Limit a ``get()`` function to a given package.
```

Reply:
I guess this one may be hard to make accurate.